brave/browser-laptop
# Torrent warning tor tabs

## Fixes:
https://github.com/brave/browser-laptop/issues/14374

## Test Plan:

1. Open https://webtorrent.io/free-torrents in a tor private tab.
2. Open any `<torrent> | <magnet>` links.
3. Verify if the warning is displayed.

<img width="925" alt="screen shot 2018-06-11 at 3 48 45 pm" src="https://user-images.githubusercontent.com/1903815/41261128-29cf52d2-6d8f-11e8-8334-65213899e5e1.png">
